library/std/src/sys/windows/process.rs

```diff
@@ -97,6 +97,14 @@ pub struct StdioPipes {
     pub stderr: Option<AnonPipe>,
 }
 
+#[derive(Debug)]
+enum Arg {
+    /// Add quotes (if needed)
+    Regular(OsString),
```

Comment:
```suggestion
    AutoQuote(OsString),
```

maybe? It adds semantic meaning.

Reply:
I think it's fine as-is. It's for the regular `arg`, not `arg_auto` or such.